tests/test_loss_auto.py

```diff
@@ -202,7 +207,7 @@ def test_multi_output_support(user_compiled, est_cls):
 def test_multiclass_single_output_unit():
     """Test that multiclass targets requires > 1 output units.
     """
-    est = KerasClassifier(model=shallow_net, model__single_output=True)
+    est = KerasClassifier(model=shallow_net, model__outputs=1)
     y = np.random.choice(N_CLASSES, size=(len(X), 1)).astype(int)
     with pytest.raises(
         ValueError,
```

Comment:
Suggested change:

``` diff
-         match="Multi-class targets require the model to have >1 output units",
+        match="Targets with {n_classes} require the model to have {n_classes} output units, not {n_outputs} outputs",
```

Reply:
I'm not sure how much this helps. There can only be 1 or >1 output units. By saying you need >1, it is clear that you must have 1, but if needed we can print `require the model to have >1 output units, but this model has only 1 output unit`. But I don't think it makes sense to make this dynamic, which would also require pulling in the `n_classes_` attribute, handling when it is not there, etc.